core/server/guilds.js

```diff
@@ -30,7 +30,10 @@ Meteor.methods({
     Meteor.users.update({ _id: { $in: userIds } }, { $set: { guildId } }, { multi: true });
 
     // analytics
-    userIds.forEach(userId => analytics.identify(Meteor.users.findOne(userId)));
+    userIds.forEach(userId => {
```

Comment:
Also probably not in the scope of this PR, but here it would be cleaner to find all the users and map the analytics on the cursor rather than fetching them one by one in a loop.
Requesting the guild for each users is also a waste.

Reply:
Indeed! I changed the code to make only one request. I'm not sure I understood your feedback about the guild, the request is outside the loop so I left the code as it was.

Reply:
Yes sorry, I misread the code, there is nothing wrong with the guild :)